### ISSUE TITLE
about reproducing the result mIoU 78.1

### ISSUE THREAD

**Issue by LordLiang** [kumuji/mix3d#5]
Hello, thank you for the nice work! 
If I want to reproduce your result in your paper, I just need to change 'voxel_size=0.02'. Is it right? Anything else should be changed?

**Comment by kumuji** [comment]
Not really, we use original MinkowskiNet repository for benchmark training. You can the [patch file](https://github.com/kumuji/mix3d/blob/master/SpatioTemporalSegmentation-ScanNet.patch) in the repo.

**Comment by LordLiang** [comment]
Thank your for your reply. By the way, I noted that you used normal information as additional feature. Can you tell me the difference between with normal and without normal？ Does your benchmark training use normal? If without normal, how much the final result can reach?

**Comment by kumuji** [comment]
We saw an improvement (around 0.5 miou) on the validation set, but not on the test set. For the final submission we decided not to use them as it just takes more time to train.
